README.md

```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://travis-ci.org/jscs-dev/node-jscs.svg?branch=master)](https://travis-ci.org/jscs-dev/node-jscs)
+﻿[![Build Status](https://travis-ci.org/jscs-dev/node-jscs.svg?branch=master)](https://travis-ci.org/jscs-dev/node-jscs)
```

Comment:
Why is this line changed? Seems no different to me? @am11, are you using correct line endings?